src/platforms/hosted/README.md

```diff
@@ -72,12 +81,14 @@ in windows. Make sure to use the `mingw64` shell from msys2, otherwise,
 you may get compilation errors. You will need to install the libusb
 and libftdi libraries, and have the correct mingw compiler.
 You can use these commands to install dependencies, and build PC-hosted BMP
-from a mingw64 shell, from within the `src` directory:
+from a mingw64 shell, from within the `blackmagic` directory:
```

Comment:
This should be `UCRT64 shell`, and should use `pacboy` - This entire passage is outdated.

You could reasonable link the [build instructions for Windows](https://black-magic.org/knowledge/compiling-windows.html#building-bmda-on-a-windows-machine) from the website as we rewrote all this there.

Reply:
Hm, I'm not comfortable with the windows workflow, do you mind writing it for me? I can apply it if you just give me an excerpt

Reply:
Sure thing, we'll review the passage fully and write a suggestion a bit later today then.